include/http/middleware/routing.js

```diff
@@ -93,7 +93,7 @@ module.exports = pb => ({
         req.pathVars = pathVars
     },
     inactiveAccessCheck: (req, res) => {
-        var inactiveSiteAccess = req.route.inactive_site_access;
+        var inactiveSiteAccess = req.route ? req.route.inactive_site_access : null;
```

Comment:
Please explain at what moment this logic reads the Inactive status of the associated TN. I could not find that in pencilblue or CMSPencilblue

Reply:
Inactive site access should only be checked when a tn is disabled in the admin console. That is either upon first creation or if the site was taken down for changes. Maintenance. Or the client did not renew. 

There’s additional logic about inactive site redirects that might cause some strange behaviors. 

Also the cause for this to be null is concerning. I’d check the collections related to site objects to see if anything is malformed. 

Hope that helps 😊

Reply:
Thanks for that @lgenet